client/payment-details/summary/index.tsx

```diff
@@ -170,8 +179,12 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 	const renderStorePrice =
 		charge.currency && balance.currency !== charge.currency;
 
+	// // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
A bit confused on this comments block, can we do a bit of cleaning here?

Reply:
It's a dead code that will be removed from PR before merging. It's used to simulate the "order missing" flow. Without the comments the TS compilers errors when simulating missing order